editor/plugins/org.fusesource.ide.project/plugin.xml

```diff
@@ -87,28 +87,6 @@
 	</extension>
 
  <extension
-       point="org.eclipse.ui.commands">

```

Comment:
I don't see another extension point for this command and handler. As the commit mentioned a "fixed double entries". I just want to check that you want to delete them. (defined in another plugin perhaps?)